### DISCUSSION TITLE
How will HA and OE cope with the clock change?

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#494] [Q&A]
General HA question really but it doesn't seem to show the extra hour around the clock change tonight.
What will the integration do with the extra hour of plunge price?

**Comment by anothermort** [comment]
It seems we get two 1-2am's. So I think we get to 2am and then just go back by rewinding an hour and do it all again.

![2023-10-28 15_54_13-Market data _ Nord Pool — Mozilla Firefox](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/82e95563-0337-499b-a9a4-5e9427de6d79)

**Reply by deltamelter** [reply]
Yea, OE and dave's integration cope just fine with the extra hour, as does the rates card.  Apex chart just tags an extra hour onto the end leaving everything after 01:30 an hour out...

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/27061628/b77739ae-cafe-47c3-bb2b-550f6ae1f72f)
![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/27061628/3c7137c7-8e54-4635-aa82-ae10e2b35e4d)
